src/Components/Bounty/index.js

```diff
@@ -1,10 +1,66 @@
-import React from "react";
+import React, { Component } from 'react';
 import "./style.css";
+import findByType from "./../../utils/findByType"
 
 // Main Bounty component, belongs in Main Display
 
-const Bounty = () => (
-    <div className="bounty">I'm the bounty Main Component!</div>
-);
+const BountyInfo = () => null;
+const BountyReward = () => null;
+const BountyTitle = () => null;
 
+class Bounty extends Component {
+
+    renderTitle() {
+        const { children } = this.props;
+        const bountyTitle = findByType(children, BountyTitle);
+
+        if (!BountyTitle) {
+            return null;
+        }
+
+        return <div className="bountyTitle">BountyTitle Rendered!{bountyTitle.props.children}</div>;
+    }
+
+    renderInfo() {
+        const { children } = this.props;
+        const bountyInfo = findByType(children, BountyInfo);
+
+        if (!bountyInfo) {
+            return null;
+        }
+
+        return <div className="bountyInfo">BountyInfo Rendered!{bountyInfo.props.children}</div>;
+    }
+
+    renderReward() {
+        const { children } = this.props;
+        const bountyReward = findByType(children, BountyReward);
+
+        if (!bountyReward) {
+            return null;
+        }
+
+        return <div className="bountyReward">BountyReward Rendered! {bountyReward.props.children}</div>
+    }
+
+    render() {
+        return (
+            <div className="bountyContainer">Bounty Rendered!
+                <div className="bountyTitleContainer">
+                    {this.renderTitle()}
+                </div>
+                <div className="bountyInfoContainer">
+                    {this.renderInfo()}
+                </div>
+                <div className="bountyRewardContainer">
+                    {this.renderReward()}
+                </div>
+            </div>
+        )
+    }
+}
+
+Bounty.BountyTitle = BountyTitle;
```

Comment:
Why are you adding properties to this react component like this? You don't want to add properties to a react component. This is not common practice. Instead, they should probably be states